packages/react/toast/src/Toast.tsx

```diff
@@ -471,18 +485,14 @@ const ToastImpl = React.forwardRef<ToastImplElement, ToastImplProps>(
                   const eventDetail = { originalEvent: event, delta };
                   if (hasSwipeMoveStarted) {
                     swipeDeltaRef.current = delta;
-                    handleAndDispatchDiscreteCustomEvent(
-                      TOAST_SWIPE_MOVE,
-                      onSwipeMove,
-                      eventDetail
-                    );
+                    dispatchCustomEvent(TOAST_SWIPE_MOVE, onSwipeMove, eventDetail, {
+                      discrete: false,
```

Comment:
`onPointerMove` doesn't fit the heuristic and it makes sense to batch these anyway.

Reply:
Great point!